core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -180,6 +180,8 @@ public synchronized void connect() throws AlluxioStatusException {
       try {
         mAddress = getAddress();
       } catch (UnavailableException e) {
+        LOG.warn("Failed to determine master RPC address ({}), retrying: {}",
```

Comment:
@aaudiber can you check if the conflict resolution is correct?

Reply:
LGTM